frontend/plugins/resources/src/components/LanguagesPage/LanguagesPage.tsx

```diff
@@ -1,12 +1,91 @@
 import React from 'react';
-import { Content, Page } from '@backstage/core';
-
-const LanguagesPage = () => (
-  <Page themeId="home">
-    <Content>
-        <div>Welcome to language page!</div>
-    </Content>
-  </Page>
-);
+import {Content, Page, Table, TableColumn, TableFilter} from '@backstage/core';
+import {DummyResponse, Edge, Resource} from "../../../../../packages/zoo-api";
```

Comment:
Did you try with just `from "zoo-api";`? It should work as it is in `package.json`